### PR TITLE
blog about Delorean to DLRN rename

### PR DESCRIPTION
**Pull request by apevec**
@derekhiggins  please check I got historical part right

@rbowen @pmkovar I'd appreciate styling/spelling/grammar corrections!

### CODE REVIEW

File: source/blog/2016-04-06-naming-is-hard-or-who-let-the-vowels-out.html.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+On [RDO community meeting](https://www.rdoproject.org/community/community-meeting/) we decided to start collecting new name proposals and came up with the [list](https://etherpad.openstack.org/p/RDO-Delorean-rename) more or less in the same time travel and BTTF theme. First proposal was [fluzo](https://es.wikipedia.org/wiki/Condensador_de_flujo) by our Spanish-speaking folks but that was dismissed after finding out there is a company with exactly that time! Next was outatime, the DeLorean time machine's license plate text, but there's actually a real movie with that name coming soon, so it was discarded to avoid confusion and possible trademark issues. Finaly, staying with the license plate idea, DLRN was choosen: written as DLRN but you can read it Delorean.
+
+The RDO community has already started work towards replacing mentions of "Delorean" in documentation, web sites, code, and repositories. Special case is references mentioning "Delorean repository" which will be renamed as "RDO trunk" to better describe it.
+
+P.S. The answer to the question in the tile is dprince! [1]
```
Comment (dprince) [inline source/blog/2016-04-06-naming-is-hard-or-who-let-the-vowels-out.html.md]: in the title :)

File: source/blog/2016-04-06-naming-is-hard-or-who-let-the-vowels-out.html.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Naming is hard or Who Let The Vowels Out
+author: apevec
+tags: openstack, trunk, packaging
+date: 2016-04-06 16:54:12 CEST
+---
+
+
+When [Derek Higgins](http://www.openstack.org/blog/2014/02/open-mic-spotlight-derek-higgins/) has [started](https://github.com/openstack-packages/DLRN/commit/496baa4eb177873ba1421c6fbd7cc7041bd584a4) the tool to build OpenStack packages based on latest upstream aka "trunk" changes, name Delorean seemed like a good choice: tool is keeping older builds so you could "go back in time" like with a [time machine](https://en.wikipedia.org/wiki/DeLorean_time_machine), it's geeky enough and there's [Irish connection](https://en.wikipedia.org/wiki/DeLorean_Motor_Company#Manufacturing_facility).
```
Comment (derekhiggins) [inline source/blog/2016-04-06-naming-is-hard-or-who-let-the-vowels-out.html.md]: The history looks fine, a couple of small suggestion, I think make it read better

When Derek Higgins started   (leave out has I think)
**the** name Delorean seem.... 
**the** tool is keeping...

**Comment by rbowen** [comment]
With the comments made, I think this is a great post. Thanks!

**Comment by apevec** [comment]
There are also comments from @chkumar246 which I need to review, please don't merge yet.

**Comment by apevec** [comment]
@rbowen ready to merge

**Comment by apevec** [comment]
@rbowen ok, now with date reset

**Pull request merged**